### PR TITLE
fix(kconfig): remove kong version from kconfig.js

### PR DESCRIPTION
**Pull request by raoxiaoyan**
### Summary
1. Remove the Kong version from `kconfig.js`
2. Retrieve the Kong version from `useInfoStore`.

### Issue reference

FTI-5557

### CODE REVIEW

File: src/composables/useGatewayFeatureSupported.ts [hunk 1]
```diff
@@ -1,13 +1,18 @@
+import { computed } from 'vue'
+import { useInfoStore } from '@/stores/info'
 import { useGatewayFeatureSupported as sharedComposable } from '@kong-ui-public/entities-shared'
 import { config } from 'config'
 
 type Arguments = Parameters<typeof sharedComposable>[0]
 
+const useInfo = useInfoStore()
+const kongVersion = computed(() => useInfo.kongVersion)
```
Comment (nekolab) [inline src/composables/useGatewayFeatureSupported.ts:9]: why do we need to wrap `useInfo.kongVersion` in `computed` here?

Comment (raoxiaoyan) [reply src/composables/useGatewayFeatureSupported.ts:9]: 'cause the method `sharedComposable` can't get original value here.

Comment (raoxiaoyan) [reply src/composables/useGatewayFeatureSupported.ts:9]: image

File: src/components/MakeAWish.vue [hunk 2]
```diff
@@ -16,12 +16,13 @@
 import { computed } from 'vue'
 import { useRoute } from 'vue-router'
 import { useI18n } from '@/composables/useI18n'
-import { config } from 'config'
+import { useInfoStore } from '@/stores/info'
 
 const route = useRoute()
 const { t } = useI18n()
+const useInfo = useInfoStore()
```
Comment (Leopoldthecoder) [inline src/components/MakeAWish.vue:23]:
```suggestion
const infoStore = useInfoStore()
```

File: src/composables/useDetailGeneralConfig.ts [hunk 3]
```diff
@@ -1,14 +1,17 @@
 import { reactive } from 'vue'
 import { config } from 'config'
+import { useInfoStore } from '@/stores/info'
 import type { KongManagerBaseEntityConfig } from '@kong-ui-public/entities-shared'
 
+const useInfo = useInfoStore()
```
Comment (Leopoldthecoder) [inline src/composables/useDetailGeneralConfig.ts:6]:
```suggestion
const infoStore = useInfoStore()
```

File: src/composables/useDocsLink.ts [hunk 4]
```diff
@@ -1,18 +1,23 @@
 import { formatVersion } from '@/utils'
 import { config, type GatewayEdition } from 'config'
 import { EntityType } from '@/types'
+import { computed } from 'vue'
+import { useInfoStore } from '@/stores/info'
+
+const useInfo = useInfoStore()
```
Comment (Leopoldthecoder) [inline src/composables/useDocsLink.ts:7]:
```suggestion
const infoStore = useInfoStore()
```

File: src/composables/useFormGeneralConfig.ts [hunk 5]
```diff
@@ -1,14 +1,17 @@
 import { reactive } from 'vue'
 import { config } from 'config'
+import { useInfoStore } from '@/stores/info'
 import type { KongManagerBaseFormConfig } from '@kong-ui-public/entities-shared'
 
+const useInfo = useInfoStore()
```
Comment (Leopoldthecoder) [inline src/composables/useFormGeneralConfig.ts:6]:
```suggestion
const infoStore = useInfoStore()
```

File: src/composables/useGatewayFeatureSupported.ts [hunk 6]
```diff
@@ -1,13 +1,18 @@
+import { computed } from 'vue'
+import { useInfoStore } from '@/stores/info'
 import { useGatewayFeatureSupported as sharedComposable } from '@kong-ui-public/entities-shared'
 import { config } from 'config'
 
 type Arguments = Parameters<typeof sharedComposable>[0]
 
+const useInfo = useInfoStore()
```
Comment (Leopoldthecoder) [inline src/composables/useGatewayFeatureSupported.ts:8]:
```suggestion
const infoStore = useInfoStore()
```

File: src/composables/useListGeneralConfig.ts [hunk 7]
```diff
@@ -1,14 +1,17 @@
 import { reactive } from 'vue'
 import { config } from 'config'
+import { useInfoStore } from '@/stores/info'
 import type { KongManagerBaseTableConfig } from '@kong-ui-public/entities-shared'
 
+const useInfo = useInfoStore()
```
Comment (Leopoldthecoder) [inline src/composables/useListGeneralConfig.ts:6]:
```suggestion
const infoStore = useInfoStore()
```

File: src/pages/overview/Overview.vue [hunk 8]
```diff
@@ -76,9 +76,10 @@ const infoStore = useInfoStore()
 
 const config = computed(() => ({
   ...infoStore.infoConfig,
+  kongVersion: infoStore.kongVersion,
```
Comment (Leopoldthecoder) [inline src/pages/overview/Overview.vue:79]: Do we need to add `kongVersion` to `config`? I think we can just use `infoStore.kongVersion` in line 82 and line 101.

Comment (raoxiaoyan) [reply src/pages/overview/Overview.vue:79]: Doesn't seem to make a difference.

**Review by Leopoldthecoder — APPROVED** [review]

**Pull request merged**